modules/management-console-agent/src/main/java/org/apache/ignite/agent/dto/cache/CacheSqlMetadata.java

```diff
@@ -208,6 +209,32 @@ public CacheSqlMetadata setIndexes(List<CacheSqlIndexMetadata> indexes) {
         return this;
     }
 
+    /** {@inheritDoc} */
+    @Override public boolean equals(Object o) {
+        if (this == o)
+            return true;
+
+        if (o == null || getClass() != o.getClass())
+            return false;
+
+        CacheSqlMetadata that = (CacheSqlMetadata) o;
+
+        return cacheName.equals(that.cacheName) &&
+            schemaName.equals(that.schemaName) &&
+            tblName.equals(that.tblName) &&
+            typeName.equals(that.typeName) &&
+            Objects.equals(keyCls, that.keyCls) &&
+            Objects.equals(valCls, that.valCls) &&
+            Objects.equals(fields, that.fields) &&
```

Comment:
`createdBySql` is not checked in `equals` and `hashCode` - is this expected?

Reply:
`createdBySql` available only in `CacheInfo`, not in `CacheSqlMetadata`